pkg/network.go

```diff
@@ -298,6 +309,7 @@ func NewConfigFromMap(data map[string]string) (*Config, error) {
 		cm.AsString(TagTemplateKey, &nc.TagTemplate),
 		cm.AsInt(RolloutDurationKey, &nc.RolloutDurationSecs),
 		cm.AsBool(AutocreateClusterDomainClaimsKey, &nc.AutocreateClusterDomainClaims),
+		cm.AsBool(EnableMeshPodAddressabilityKey, &nc.EnableMeshPodAddressability),
 	); err != nil {
 		return nil, err
 	}
```

Comment:
As `EnableMeshPodAddressability` is only work with net-istio, should we add a validation for the class like `EnableMeshPodAddressability && nc.DefaultIngressClass != IstioIngressClassName`
?

Reply:
Hmmmm, I **do** see your point but I'm wondering if it'll hamper testing of the feature. You could for example imagine of people using istio for some services and not as the default and we'd still want to set this I believe.

On the other hand, I don't see people misusing this just yet :thinking:. I think I'd lean towards keeping it open for now.

Reply:
I understand and that makes sense. Let's keep as it is.